src/main/java/uk/gov/hmcts/reform/pip/data/management/errorhandling/GlobalExceptionHandler.java

```diff
@@ -67,8 +68,18 @@
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
     }
 
-    @ExceptionHandler(EmptyRequestHeaderException.class)
-    public ResponseEntity<ExceptionResponse> handle(EmptyRequestHeaderException ex) {
+    @ExceptionHandler(HeaderValidationException.class)
+    public ResponseEntity<ExceptionResponse> handle(HeaderValidationException ex) {
+
+        ExceptionResponse exceptionResponse = new ExceptionResponse();
+        exceptionResponse.setMessage(ex.getMessage());
+        exceptionResponse.setTimestamp(LocalDateTime.now());
+
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
+    }
+
+    @ExceptionHandler(ValidationException.class)
+    public ResponseEntity<ExceptionResponse> handle(ValidationException ex) {
```

Comment:
can we merge these 2 in the hanlder so we only handle a validation exception and the `HeaderValidationException` extends the `ValidationException`?

Reply:
Updated :)